packages/dashboard/src/msw/iot-sitewise/handlers/listAssetProperties/listAssetProperties.ts

```diff
@@ -12,8 +12,11 @@ export function listAssetPropertiesHandler() {
       return res(ctx.status(404));
     }
 
+    const properties = asset.assetProperties;
+    asset.assetCompositeModels?.forEach((model) => model.properties?.forEach((property) => properties?.push(property)));
+
     const response: ListAssetPropertiesResponse = {
-      assetPropertySummaries: asset.assetProperties,
+      assetPropertySummaries: properties,
```

Comment:
Instead of mutating the properties array, could you create a new array (i.e., `assetPropertySummaries`) which you push both `assetProperties` and all of the asset composite model properties? With the current format, it's possible you are mutating the asset directly and will append all the composite model properties to `asset.assetProperties` each time the handler is called for a specific asset.